lisa/tools/lsblk.py

```diff
@@ -65,6 +65,10 @@ class DiskInfo(object):
     type: str = ""
     partitions: List[PartitionInfo] = field(default_factory=list)
 
+    @classmethod
+    def _freebsd_tool(cls) -> Optional[Type[Tool]]:
```

Comment:
Why does the disk info need this method?

Reply:
This was an error. Fixed